fili-core/src/main/java/com/yahoo/bard/webservice/web/ErrorMessageFormat.java

```diff
@@ -162,7 +163,11 @@
 
     MISSING_JOB_ID("Bard experienced an internal error. Sorry.", "Missing id for job row %s"),
 
-    INVALID_ASYNC_AFTER( "Invalid 'asyncAfter' parameter: '%s'. 'asyncAfter' must be either 'never' or an integer number of milliseconds.");
+    INVALID_ASYNC_AFTER( "Invalid 'asyncAfter' parameter: '%s'. 'asyncAfter' must be either 'never' or an integer number of milliseconds."),
+
+    FILTER_JOBFIELD_UNDEFINED("Filter JobField '%s' does not exist."),
+
+    JOBFIELD_NOT_PRESENT_IN_JOB_META_DATA("JobField '%s' is not a part of job meta data. The possible fields to filter on are '%s'");
```

Comment:
FILTER_JOBFIELD_UNDEFINED means if the user tries to filter on some weird JobField that does not exist in the system. Let's say the user says filter jobs by userTeam or something which is not a valid field.

JOBFIELD_NOT_PRESENT_IN_JOB_META_DATA means that the Field is a valid field in the system like dateUpdated(it is a part of `DefaultJobField`)but it is not a part of the JobRow. I am not sure if we will ever have this scenario.

May be we should treat both these scenarios as the same error. 

Thoughts @archolewa @cdeszaq ?